servers/rendering/renderer_rd/shaders/canvas.glsl

```diff
@@ -611,6 +611,9 @@ void main() {
 		}
 
 		light_blend_compute(light_base, light_color, color.rgb);
+#ifdef MODE_LIGHT_ONLY
+		ligth_only_alpha += light_color.a;
```

Comment:
I'm not sure what should be used here, sum, average or something else. And is `clamp` needed here?